source/administration/changelog.md

```diff
@@ -4,6 +4,133 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+   - ``group channel disable`` to disable group constraint on the specified channel.
+   - ``group channel list`` to list the groups associated with a channel.
+   - ``group channel status`` to show the group constraint status of the specified channel.
+ 
+#### Administration
+ - Added support for running two Mattermost instances on the same domain using subpaths.
+ - Added support for importing threads from Slack.
+
+### Bug Fixes
+ - Fixed an issue where releasing a mouse click while the cursor was outside of the Rename Channel modal would close the modal.
+ - Fixed an issue where a whitepage occured after uploading a plugin with an invalid ``settings_schema`` value.
+ - Fixed an issue where the announcement banner overlapped channel content.
+ - Fixed an issue where license expiration notice banner could not be dismissed prior to the license expiration date.
+ - Fixed an issue where the channel switcher autocomplete didn't function properly when autocompleting the name of a person who was the first person named in a group message channel.
+ - Fixed an issue where inline images in markdown preview didn't get expanded.
+ - Fixed an issue where replies to the parent post were not left-aligned.
+ - Fixed an issue where the timezone picker dropdown closed when trying to drag the scrollbar.
+ - Fixed an issue where the ``ExperimentalPrimaryTeam`` config.json setting no longer hid the "Leave Team" option.
+
+### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under ``"PluginSettings":`` in ``config.json``:
+   - Added ``"EnableHealthCheck": true``, to ensure all plugins are periodically monitored, and restarted or deactivated based on their health status.
+ - Under ``"NotificationLogSettings":`` in ``config.json``:
+   - Added ``"EnableConsole": true``, ``"ConsoleLevel": "DEBUG"``, ``"ConsoleJson": true``, ``"EnableFile": true``, ``"FileLevel": "INFO"``, ``"FileJson": true``, and ``"FileLocation": ""``, to implement a structured logger to keep track of push notifications.
+ - Under ``"ServiceSettings":`` in ``config.json``:
+   - Added ``"EnableBotAccountCreation": false`` to enable bot account creation.
+   - Added ``"DisableBotsWhenOwnerIsDeactivated": true`` to disable bots automatically when the owner is deactivated.
+   - Added ``"TrustedProxyIPHeader": []``, to explicitly define which IP headers are trusted.
+
+### Database Changes
+ - 
+
+### API Changes
+ - Updated API to use gziphandler wrapper if server is configured to use gzip. This ensures that the Mattermost server can respond to REST API requests with compressed data (via gzip) to reduce the amount of bandwidth used.
+ - Added an API command to create and delete a team and channel.
```

Comment:
This is actually for Cypress end-to-end testing only, not for interacting with Mattermost https://github.com/mattermost/mattermost-webapp/pull/2601#issuecomment-498448815